src/Compilers/CSharp/Portable/FlowAnalysis/LocalDataFlowPass.cs

```diff
@@ -91,11 +91,11 @@ protected int VariableSlot(Symbol symbol, int containingSlot = 0)
         /// <summary>
         /// Force a variable to have a slot.  Returns -1 if the variable has an empty struct type.
         /// </summary>
-        protected int GetOrCreateSlot(Symbol symbol, int containingSlot = 0)
+        protected int GetOrCreateSlot(Symbol symbol, int containingSlot = 0, bool createIfMissing = true)
```

Comment:
>createIfMissing [](start = 82, length = 15)

It feels like the "createIfMissing" parameter is specific to tuples, but the name doesn't reflect the fact. This might be confusing to consumers. For example, one might assume that, unless we already allocated a slot, this function won't allocate it when false is passed for ```createIfMissing``` parameter. However, it doesn't look like this is the case.